build.sbt

```diff
@@ -1,18 +1,26 @@
 import Util._
 import Dependencies._
 import Scripted._
-// import com.typesafe.tools.mima.core._, ProblemFilters._
+import com.typesafe.tools.mima.core._, ProblemFilters._
 
 def baseVersion = "1.0.0-X21-SNAPSHOT"
 def internalPath = file("internal")
 
 lazy val compilerBridgeScalaVersions = List(scala212, scala211, scala210)
 
+def mimaSettings: Seq[Setting[_]] = Seq(
+  mimaPreviousArtifacts := Set(
+  (organization.value % moduleName.value % "1.0.0-X20").cross(
+    if (crossPaths.value) CrossVersion.binary else CrossVersion.disabled)
+  )
+)
+
 def commonSettings: Seq[Setting[_]] = Seq(
   scalaVersion := scala212,
   // publishArtifact in packageDoc := false,
   resolvers += Resolver.typesafeIvyRepo("releases"),
   resolvers += Resolver.sonatypeRepo("snapshots"),
+  resolvers += Resolver.sonatypeRepo("staging"),
```

Comment:
Oh, no, actually this is necessary for now until Scala 2.12.3 has hit Central. I will remove it then.